simulation/dynamic_model_quad_diag.cpp

```diff
@@ -76,17 +76,12 @@ Dynamic_model_quad_diag::Dynamic_model_quad_diag(Servo& servo_rear_left,
     dt_s_(0.004f)
 {
     // Init local position
-    local_position_.pos[0]  = 0.0f;
-    local_position_.pos[1]  = 0.0f;
-    local_position_.pos[2]  = 0.0f;
-    local_position_.heading = 0.0f;
-    local_position_.origin.latitude         = config_.home_coordinates[0];
-    local_position_.origin.longitude        = config_.home_coordinates[1];
-    local_position_.origin.altitude         = config_.home_coordinates[2];
-    local_position_.origin.heading          = 0.0f;
+    local_position_[0]  = 0.0f;
+    local_position_[1]  = 0.0f;
+    local_position_[2]  = 0.0f;
 
     // Init global position
-    global_position_ = coord_conventions_local_to_global_position(local_position_);
+    coord_conventions_local_to_global_position(local_position_, config_.origin, global_position_);
```

Comment:
we shouldnt use the config for origin there ?

if origin changes we will start to have some weird behaviors